pkg/querier/queryrange/views.go

```diff
@@ -266,10 +268,14 @@ func (v *MergedSeriesResponseView) Materialize() (*LokiSeriesResponse, error) {
 	mat := &LokiSeriesResponse{}
 	err := v.ForEachUniqueSeries(func(series *SeriesIdentifierView) error {
 		identifier := logproto.SeriesIdentifier{Labels: make(map[string]string)}
-		series.ForEachLabel(func(name, value string) error {
+		err := series.ForEachLabel(func(name, value string) error {
 			identifier.Labels[name] = value
 			return nil
 		})
+		if err != nil {
+			return err
```

Comment:
Wrap the error please

Reply:
Done. Although I'm not quite sure it is very useful in this case.